dwave/system/composites/tiling.py

```diff
@@ -39,25 +39,40 @@
 
 
 class TilingComposite(dimod.Sampler, dimod.Composite, dimod.Structured):
-    """Composite to tile a small problem across a Chimera-structured sampler.
-
-    Enables parallel sampling for small problems (problems that are minor-embeddable in
-    a small part of a D-Wave solver's :term:`Chimera` graph).
-
-    Notation *CN* refers to a Chimera graph consisting of an NxN grid of unit cells, where
-    each unit cell is a bipartite graph with shores of size t. The D-Wave 2000Q QPU
-    supports a C16 Chimera graph: its 2048 qubits are logically mapped into a 16x16 matrix of
-    unit cell of 8 qubits (t=4).
-
-    A problem that can be minor-embedded in a single unit cell, for example, can therefore
-    be tiled across the unit cells of a D-Wave 2000Q as 16x16 duplicates. This enables
-    sampling 256 solutions in a single call.
+    """Composite to tile a small problem across a sampler.
+
+    Enables parallel sampling for small problems (problems that are 
+    minor-embeddable in a :term:`Chimera` graph of dimensions (``sub_m``, ``sub_n``, ``t``). 
+    The sampler used can be Chimera or Pegasus structured, since Chimera graphs
+    are subgraphs of Pegasus graphs.
+
+    Notation *CN* refers to a Chimera graph consisting of an NxN grid of unit 
+    cells, where each unit cell is a bipartite graph with shores of size t. 
+    The D-Wave 2000Q QPU supports a C16 Chimera graph: its 2048 qubits are 
+    logically mapped into a 16x16 matrix of unit cell of 8 qubits (t=4).
+
+    Notation *PN* referes to a Pegasus graph consisting of 3x(N-1)x(N-1) grid 
+    of cells, where each unit cell is a bipartite graph with shore of size t, 
+    supplemented with odd couplers (see nice_coordinate definition). The 
+    Advantage QPU supports a P16 Pegasus graph: It's qubits may be mapped to 
+    3x15x15 matrix of unit cells each of 8 qubits. This code supports tiling of 
+    chimera-structured problems, with an option of additional odd-couplers,
+    onto Pegasus. 
+
+    A problem that can be minor-embedded in a single chimera unit cell, for 
+    example, can therefore be tiled across the unit cells of a D-Wave 2000Q as 
+    16x16 duplicates (or Advantage as 3x15x15 duplicates), subject to solver
+    yield. This enables up to 256 (625) parallel samples per read.
 
     Args:
-       sampler (:class:`dimod.Sampler`): Structured dimod sampler such as a :class:`~dwave.system.samplers.DWaveSampler()`.
-       sub_m (int): Number of rows of Chimera unit cells for minor-embedding the problem once.
-       sub_n (int): Number of columns of Chimera unit cells for minor-embedding the problem once.
-       t (int, optional, default=4): Size of the shore within each Chimera unit cell.
+       sampler (:class:`dimod.Sampler`): Structured dimod sampler such as a 
+           :class:`~dwave.system.samplers.DWaveSampler()`.
+       sub_m (int): Minimum number of Chimera unit cell rows required for
+           minor-embedding without duplication.
```

Comment:
"required for minor-embedding without duplication" makes perfect sense and yet I did not understand the meaning right away. Perhaps "required for minor-embedding a single instance of the problem"?